fighthealthinsurance/websockets.py

```
@@ -54,6 +55,59 @@ def _get_client_ip_from_scope(scope: Optional[dict] = None) -> Optional[str]:
+    """
+    persisted_count = -1
+    denial_attempts: Optional[int] = None
+    try:
+        if denial_id is not None:
+            persisted_count = await ProposedAppeal.objects.filter(
+                for_denial__denial_id=denial_id
+            ).acount()
+            denial = await Denial.objects.filter(denial_id=denial_id).afirst()
+            if denial is not None:
+                denial_attempts = denial.gen_attempts
+    except Exception as lookup_error:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)